colcon-build/action.yaml

```diff
@@ -75,6 +79,9 @@ runs:
         else
             rosdep update
         fi
+        if [ -f "${{ inputs.underlay-workspace }}/setup.sh" ]; then
+          . ${{ inputs.underlay-workspace }}/setup.sh
```

Comment:
```suggestion
          source ${{ inputs.underlay-workspace }}/setup.sh
```

To keep it consistent.

Reply:
Since original code uses `.` https://github.com/autowarefoundation/autoware-github-actions/blob/main/colcon-build/action.yaml#L89, I also use it.

Reply:
Fair enough 👍